File: Go/01.Go-Basic-February-2022/05.conditionalStatementAdvanced-Lab/04.personalTitles.go
package main

import "fmt"

func main() {
	var age float32 = 12
	// fmt.Scanln(&age)
	var gender string = "m"
	// fmt.Scanln(&gender)

	if gender == "m" {
		if age < 16 {
			fmt.Print("Master")
		} else {
			fmt.Print("Mr.")
		}
	} else if gender == "f" {
		if age < 16 {
			fmt.Print("Miss")
		} else {
			fmt.Print("Ms.")
		}
	}
}

File: JavaScript/01.JavaScript-Basic-February-2021/05.conditionalStatementsAdvanced-Lab/04.personalTitles.js
function title(input) {
    let age = Number(input[0]);
    let gender = input[1];

    if (gender == "f") {
        if (age < 16) {
            console.log("Miss");
        } else {
            console.log("Ms.");
        }
    } else {
        if (age < 16) {
            console.log("Master");
        } else {
            console.log("Mr.");
        }
    }
}

title(["12", "f"]);
title(["17", "m"]);
title(["25", "f"]);
title(["13.5", "m"]);